### ISSUE TITLE
Password visibility

### ISSUE THREAD

**Issue by bartech** [woocommerce/woocommerce#25624]
**Describe the bug**
On password reset form after clicking on eye icon to show password, input type is correctly changed to type text but clicking again don't toggle input type back to password

**To Reproduce**
1. Go to lost password page and reset password (fill form and sand)
2. Click on link in email
3.  On reset password page form fill in password field
4. Click eye icon to show password
5. Click again to hide password

**Expected behavior**
Password show be hidden again

**Isolating the problem (mark completed items with an [x]):**
Problem is code in `woocommerce.js` responsible for targeting password field. It's trying to target elem by `input[type="password"]` or `input[name="password"]` in reset password form input names for password fields are `password_1` and `password_2`. Click event is working only once because targeting by `input[type="password"]` works but after first click input is changed to `input[type="text"]` and second selector via name attribute do not match.

**Solution**
Change jQuery selector from  `input[name="password"]` to  `input[name^="password"]`

**Comment by juliaamosova** [comment]
Hi @bartech101,

Thank you for taking the time reporting this bug, we really appreciate your help. I can reproduce it on my end. 

It is possible to click on the eye icon to view the password but when clicking again to hide it, the password doesn't get hidden:

![pw](https://user-images.githubusercontent.com/19143190/74161804-78a10a00-4c17-11ea-9b23-2c869c5300ef.jpg)

We won’t be able to include this fix in the upcoming release due to the lower priority of this issue compared to others reported. We’re going to add it to our backlog so we can include it in our planning for one of our future releases.

**Comment by juliaamosova** [comment]
@bartech101 you are welcome to submit a PR if you like as well.
